shuup/core/models/_attributes.py

```diff
@@ -119,6 +124,15 @@ class Attribute(TranslatableModel):
             "Attributes can be shown on the product detail page or can be used to enhance product search results."
         ),
     )
+    ordering = models.IntegerField(
+        default=0,
+        help_text=_("The ordering in which your attribute will be displayed."),
+    )
+    max_choices = models.PositiveIntegerField(
+        default=1,
+        verbose_name=_("max choices"),
+        help_text=_("Amount of choices that user can choose from existing options."),
```

Comment:
add a text saying this will be only used when the type of attribute is choices. maybe we can also create a min_choices which we can use to force users to select a certain number of options?